django/db/backends/oracle/schema.py

```diff
@@ -43,7 +43,9 @@ def quote_value(self, value):
     def remove_field(self, model, field):
         # If the column is an identity column, drop the identity before
         # removing the field.
-        if self._is_identity_column(model._meta.db_table, field.column):
+        if field.column and self._is_identity_column(
+            model._meta.db_table, field.column
+        ):
```

Comment:
I'm not sure about this. Ultimately, it's a question of whether the `column_name` is a `str`, a `str | None` or a `Field`.

1. At the moment it's a `str`.
2. We could check if `column_name is None` inside the function and it would be `str | None`.
3. Or we could change it to `Field` as you suggested.

I personally like the idea that it's always a `str` and that the calling code has to respect the type.

Reply:
Part of my suggestion would be to rename `column_name` to `field` or `column` (depending whether you pass `field` or `field.column`). At this point I'm leaning to your version 2 with a rename to `column`. But maybe other reviewers have opinions here?

Reply:
Alright, let's do it that way then, I pushed the change.